docs/source/kedro_project_setup/configuration.md

```diff
@@ -119,7 +119,35 @@ kedro run --env=test
 
 If no `env` option is specified, this will default to using the `local` environment to overwrite `conf/base`.
 
-If, for some reason, your project does not have any other environments apart from `base`, i.e. no `local` environment to default to, you must customise `KedroContext` to take `env="base"` in the constructor and then specify your custom `KedroContext` subclass in `src/<package_name>/settings.py` under the `CONTEXT_CLASS` key.
+If, for some reason, your project does not have any other environments apart from `base`, i.e. no `local` environment to default to, you must customise the configuration loader you're using to take `default_run_env="base"` in the constructor and then specify your custom config loader subclass in `src/<package_name>/settings.py` under the `CONFIG_LOADER_CLASS` key.
+Below is an example of such a custom class. If you're using the `TemplatedConfigLoader` or the `OmegaConfigLoader` you need to use either of those as the class you are subclassing.
+
+```python
+from kedro.config import ConfigLoader
```

Comment:
I know it might look a bit repetitive but is it worth adding what file are we talking about here?

Initially I thought it was `settings.py`, but then I saw that the `settings.py` is the one below this code snippet, and therefore the custom class is being defined elsewhere. Maybe we can give a suggestion of where this file can live?

Reply:
Done!